diff --git a/lib/input.js b/lib/input.js
index 4b5ee5e02f8ecdf880594d191ec1e84b983faeaa..f0d0e5ee36efb9ca996dec6f9dc7cd870ee4427b 100644
--- a/lib/input.js
+++ b/lib/input.js
@@ -1,8 +1,8 @@
 'use strict'
 
 let { SourceMapConsumer, SourceMapGenerator } = require('source-map-js')
-let { fileURLToPath, pathToFileURL } = require('url')
-let { isAbsolute, resolve } = require('path')
+let { fileURLToPath, pathToFileURL } = require('node:url')
+let { isAbsolute, resolve } = require('node:path')
 let { nanoid } = require('nanoid/non-secure')
 
 let terminalHighlight = require('./terminal-highlight')
diff --git a/lib/map-generator.js b/lib/map-generator.js
index 523b46397ed258ca3bac0ce6ef25e909c189253c..ab6aa7d8db9d0ca8485a5db6208f0610ffb64631 100644
--- a/lib/map-generator.js
+++ b/lib/map-generator.js
@@ -1,8 +1,8 @@
 'use strict'
 
 let { SourceMapConsumer, SourceMapGenerator } = require('source-map-js')
-let { dirname, relative, resolve, sep } = require('path')
-let { pathToFileURL } = require('url')
+let { dirname, relative, resolve, sep } = require('node:path')
+let { pathToFileURL } = require('node:url')
 
 let Input = require('./input')
 
diff --git a/lib/previous-map.js b/lib/previous-map.js
index f3093dfb8e77e305a648ae72a8062cac759e81e9..f06ff92e7a836d8714850ea7a32b3ba35ba6ecce 100644
--- a/lib/previous-map.js
+++ b/lib/previous-map.js
@@ -1,8 +1,8 @@
 'use strict'
 
 let { SourceMapConsumer, SourceMapGenerator } = require('source-map-js')
-let { existsSync, readFileSync } = require('fs')
-let { dirname, join } = require('path')
+let { existsSync, readFileSync } = require('node:fs')
+let { dirname, join } = require('node:path')
 
 function fromBase64(str) {
   if (Buffer) {
